llvm/lib/SYCLLowerIR/SYCLPropagateJointMatrixUsage.cpp

```diff
@@ -0,0 +1,244 @@
+//===---- SYCLPropagateJointMatrixUsage.cpp - SYCLPropagateJointMatrixUsage Pass
+//--===//
```

Comment:
```suggestion
//===---- SYCLPropagateJointMatrixUsage.cpp -------------------------------===//
```

I would maybe suggest to drop this small "description" completely since it doesn't fit.
You can put it on the next line, but considering that it doesn't bring a lot of extra information, I think that we can just drop it.